hashing/chaining/Chaining_HashTable.java

```diff
@@ -0,0 +1,135 @@
+package hashing.chaining;
+
+import java.util.LinkedList;
+import java.util.Scanner;
+
+
+public class Chaining_HashTable {
+    LinkedList<Pair> arr[];
+    int m,n;
+    int collisions;
+    class Pair{
+        int key;
+        int val;
+        public Pair(int k,int v){
+            key=k;
+            val=v;
+        }
+    }
+    public Chaining_HashTable(Integer m){
+        this.m=m;
+        this.n=n;
+        collisions=0;
+        //creating bucket of size m
+        arr=new LinkedList[m];
+    }
+    public int getCollisions(){
+        int bucketElements=0;
+        // total no. of collision is equal to the 
+        //sibmisiion of size of all linkedList(chains) excluding first element
```

Comment:
Please correct this typo.
sibmisiion -> summation

Reply:
Okay Sir, I have made the changes.